spinalcordtoolbox/scripts/sct_dmri_separate_b0_and_dwi.py

```diff
@@ -205,7 +211,13 @@ def main(argv=None):
     # Average DWI images
     if average:
         printv('\nAverage DWI...', verbose)
-        run_proc(['sct_maths', '-i', dwi_name + ext, '-o', dwi_mean_name + ext, '-mean', 't'], verbose)
+        img = Image(dwi_name + ext)
+        out = img.copy()
+        dim_idx = 3
+        if len(np.shape(img.data)) < dim_idx + 1:
+            raise ValueError("Expecting image with 4 dimensions!")
```

Comment:
Is this the right check for `-mean t`? See the previous review comment, which has a different check for `-mean t`.